sdk/storage/storage-blob/src/BlobClient.ts

```diff
@@ -50,19 +57,29 @@ export interface BlobDownloadOptions {
    */
   abortSignal?: AbortSignalLike;
   /**
-   * Optional. The version string of the snapshot to download.
+   * An opaque DateTime string value that, when present, specifies the blob snapshot to retrieve.
    *
    * @type {string}
    * @memberof BlobDownloadOptions
    */
   snapshot?: string;
   /**
-   * Specifies whether to retrieve the MD5 hash of the range when downloading a range of bytes.
+   * When this is set to true and download range of blob, the service returns the MD5 hash for the range,
+   * as long as the range is less than or equal to 4 MB in size.
+   *
+   * rangeGetContentCrc64 and rangeGetContentMD5 cannot be set at same time.
    *
    * @type {boolean}
    * @memberof BlobDownloadOptions
    */
   rangeGetContentMD5?: boolean;
+  /**
+   * When this is set to true and download range of blob, the service returns the CRC64 hash for the range,
+   * as long as the range is less than or equal to 4 MB in size.
+   *
+   * rangeGetContentCrc64 and rangeGetContentMD5 cannot be set at same time.
+   */
```

Comment:
Minor: add the @type and @memberof info, I find these are added for track 2, would better to make this consistent if this is convention for track 2.

Reply:
Done.